xocto/localtime.py

```diff
@@ -68,16 +72,12 @@ def datetime(
     minute: int = 0,
     second: int = 0,
     microsecond: int = 0,
-    is_dst: bool | None = None,
 ) -> datetime_.datetime:
     """
     Return a datetime in the local timezone.
-
-    A boolean value is required for is_dst to unambiguously determine the appropriate dt during the
-    window when the hour goes back.
     """
     dt = datetime_.datetime(year, month, day, hour, minute, second, microsecond)
-    return timezone.make_aware(dt, is_dst=is_dst)
+    return timezone.make_aware(dt)
```

Comment:
Would it make sense to retain the `is_dst` parameter and convert it to the appropriate `fold` value on the `datetime` constructor?

Reply:
Unfortunately that's not quite straight forward as the two don’t map onto one another perfectly.

https://pytz-deprecation-shim.readthedocs.io/en/latest/migration.html#semantic-differences-between-is-dst-and-fold